runtime-modules/proposals/codex/src/types.rs

```diff
@@ -118,6 +119,9 @@ pub enum ProposalDetails<
 
     /// `Update Channel Payouts` proposal
     UpdateChannelPayouts(UpdateChannelPayoutsParameters),
+
+    /// `Freeze Palet` proposal
+    FreezePallet(bool, FrezablePallet),
```

Comment:
Since there is only one proposal to do two actions, a better name would be (like the dispatch call) `SetPalletFozenStatus` ?